app/views/posts/index.html.erb

```diff
@@ -1,42 +1,53 @@
-<main>
-  <section>
-    <div class="card pagination">
-      <div class="user-image border">
-        <%= image_tag(@user.photo, class: "add-css-class") %>
-      </div>
-      <div class="user-data border">
-        <h1><%= @user.name %></h1>
-        <p class="number-posts">Number of posts: <%= @user.post_counter %></p>
-        <div class="right">
-          <%= link_to "Add a post",new_user_post_path(@user), class: "btn right" %>
+<div class="bg-dark min-vh-100 text-white container pt-5" style="">
+
+  <%= render 'users/user' , user: @user %>
+
+    <% @user.all_posts.each do |post| %>
```

Comment:
- [ ] Once again, kindly go through your app and ensure the app works at least and avoid unnecessary error. This `all_posts` is not defined anywhere in your app, but you are calling it as if it was defined somewhere.

  ![image](https://github.com/Marcraphael12/rails-blog-app/assets/36542986/5d8cedac-92d1-490c-adc2-bc8c550f9edd)